app/lib/server/functions/notifications/mobile.js

```diff
@@ -62,6 +62,7 @@ export async function getPushData({ room, message, userId, senderUsername, sende
 			name: room.name,
 			messageType: message.t,
 			tmid: message.tmid,
+			...message.t === 'e2e' && { msg: message.msg },
```

Comment:
@rodrigok is possible to us to send the username/name based on `UI_Use_Realname` setting to be joined with the message after decrypted on client side?
When it came from a group we should show `(username/name): message`, it's a little pain to read the setting of `Realname` on notification context for us, so if it's possible to be sent by the server, it would be very good.